src/main/resources/org/jenkinsci/plugins/discardbuild/DiscardBuildPublisher/help.jelly

```diff
@@ -0,0 +1,92 @@
+            </p>
+        </dd>
+    </dl>
+    <dl>
+        <dt><code>Interval to keep old builds (by days or # of builds)</code></dt>
+        <dd>
+            <p>
+                A specialized setting allowing a user to request that builds seperated by a declared interval of age
+                or build quantity are always kept. For e.g. setting day intervals to #15 would ensure builds #1, #16,
+                #31, etc are kept. Please test this setting carefully to ensure the desired functionality is produced.
+            </p>
+        </dd>
+    </dl>
+    <dl>
+        <dt><code>Status to discard</code></dt>
+        <dd>
+            <p>
+                Toggles which build status' the plugin includes in its discards. If no settings are checked, the plugin
+                will clear all build status' the same. Recommended if you don't want to keep particular build outcomes.
+            </p>
+        </dd>
+    </dl>
+    <dl>
+        <dt><code>Min/Max Logfile size in bytes</code></dt>
+        <dd>
+            <p>
+                Sets a file size in bytes (B) that log files must remain under to be kept in the build history. If
+                this size is exceeded, these builds will be discarded.
+            </p>
+        </dd>
+    </dl>
+    <dl>
+        <dt><code>Regular expression</code></dt>
+        <dd>
+            <p>
+                Parses log files and discards those that contain a user-defined expression.
+            </p>
+        </dd>
+    </dl>
+    <dl>
+        <dt><code>Keep last builds</code></dt>
+        <dd>
+            <p>
+                Removes last builds from build history array prior to being passed through any discard conditions.
+            </p>
+        </dd>
+    </dl>
+    <dl>
+        <dt><code>Require both quantity and age conditions to be met for build discard</code></dt>
+        <dd>
+            <p>
+                Allows a user to request both quantity and age conditions be met prior to build discard.
+                This means that if N max builds are set with M max age, builds will only be cleared if they exceed
+                the given age <i>and</i> if their quantity exceed the N max build quantity. The feature will thus never
+                allow build deletion to leave less builds than the N max builds set, or allow build deletion of builds
+                under the M max age <i>regardless</i> of N max builds set.
+            </p>
+            <p>
+                <b>NOTE:</b> This feature does nothing if <i>Days to keep builds</i> and/or <i>Max # of builds to keep</i> are empty.
+            </p>
+            <p>
+                <b>Recommended Configuration:</b> Set a max age and build quantity and turn this feature on, leave
+                additional plugin settings in their default off setting. This configuration has been tested and
+                functions as described in the feature description.
```

Comment:
We usually do not say that the feature has been tested. If it were untested then yes it would be important to mention it.